firmware/components/driver_display_hub75/font_6x3.c

```diff
@@ -9,6 +9,110 @@
 
 #ifdef CONFIG_DRIVER_HUB75_ENABLE
 
+typedef struct character {
+    uint8_t width; /* Characters limited to 255 pixels wide */
+    uint8_t rows[6]; /* 6 pixels tall */
+} character;
+
+static const character font_6x3[] = {
```

Comment:
The original reason for having the fonts in header files was to allow sharing them across components. This is no longer needed, so I'm fine with pushing it to the C files. I'm interested to hear your reason for moving it though!

Reply:
I wrote my rational in the commit message:
https://github.com/badgeteam/ESP32-platform-firmware/pull/233/commits/f415f753fa58041c6e776b012c75ff4ab1c5858a

In fact, I first added the `const` keyword to make sure it stays in flash, that then lead to a linker error since the header was included twice. That is when I moved it. In theory the `const` keyword should make sure the array doens't land in memory and stays in flash. I compared memory/flash usage before and after the change and it did not change, so I guess the compiler realized that this is only read from and left it in flash already.